forage-android/src/main/java/com/joinforage/forage/android/network/data/CheckBalanceRepository.kt

```diff
@@ -104,16 +102,19 @@ internal class CheckBalanceRepository(
             }
 
             if (attempt == MAX_ATTEMPTS) {
-                logger.debug("Max attempts reached. Returning last response")
+                logger.e("Max polling attempts reached for balance check of Payment Method $paymentMethodRef")
```

Comment:
include `contentId` in log? Also, worth including `paymentMethodRef` in log tag / Context for easier DataDog search?